en/sensor_bus/README.md

```diff
@@ -1 +1,3 @@
-# Sensor and Actuator Busses
\ No newline at end of file
+# Sensor and Actuator I/O
+
+This section contains topics about integrating sensors and actuators into PX4. It covers both sensor busses ([I2C](../sensor_bus/i2c.md), [UAVCAN ](../uavcan/README.md), [UART](../sensor_bus/uart.md), SPI) and also the main PWM outputs.
```

Comment:
@bkueng 
1. Re "also the main PWM outputs."
   - Is this the right/best name for the AUX/MAIN ports? (probably should change to I/O rather than outputs too)
   - What I/O is typically run on this port? Motor control obviously and other actuators (e.g. parachutes). But what about input?
   - What are the mechanics of integrating a sensor on these ports. Specifically, say I want to add X sensor to an AUX port, am I again writing a device driver that reads the inputs and outputs uORB topics?
   - What about actuators - say I want to open a brand new type of gripper?

Reply:
- You could name it 'PWM ports'
- input: Lidar Lite, or some ultrasonic sensors.
- Basically yes, but I think it's somewhat involved. I think it's the easiest to point to an existing driver
- actuators are simpler: you choose an actuator controls topic and then configure the mixer accordingly (I think it's documented on the mixer page).

Reply:
Thank you. I've updated to "PWM ports" and taken notes about what the associated topic will have to say in https://github.com/PX4/Devguide/issues/387#issuecomment-353212719

I will also merge on the assumption this is otherwise OK.